lib/ProMotion/screen/screen_module.rb

```diff
@@ -25,41 +25,6 @@ def modal?
       self.modal == true
     end
 
-    def resolve_title
```

Comment:
Made these methods private, like they should have been. No other changes, just moved them to the private method section.